conf/up4.bess

```diff
@@ -0,0 +1,487 @@
+                              {'value_int':0},               # enb_teid
+                              {'value_int':start_ue_ip+i},   # ueaddr ip
+                              {'value_int':0},               # inet ip
+                              {'value_int':0},               # ue port
+                              {'value_int':0},               # inet port
+                              {'value_int':0}],              # proto-id
+                      masks=[{'value_int':0xFF},	     # src_iface-mask
+                              {'value_int':0},		     # tunnel_ipv4_dst-mask
+                              {'value_int':0},               # enb_teid-mask
+                              {'value_int':0xFFFFFFFF},      # ueaddr ip-mask
+                              {'value_int':0},               # inet ip-mask
+                              {'value_int':0},               # ue port-mask
+                              {'value_int':0},               # inet port-mask
+                              {'value_int':0}])              # proto-id-mask
+
+        PDRLookup.delete(values=[{'value_int':Access},	     # src_iface
+                              {'value_int':0},		     # tunnel_ipv4_dst
+                              {'value_int':0},               # enb_teid
+                              {'value_int':0},               # inet ip
+                              {'value_int':start_ue_ip+i},   # ueaddr ip
+                              {'value_int':0},               # inet port
+                              {'value_int':0},               # ue port
+                              {'value_int':0}],              # proto-id
+                      masks=[{'value_int':0xFF},	     # src_iface-mask
+                              {'value_int':0},		     # tunnel_ipv4_dst-mask
+                              {'value_int':0},               # enb_teid-mask
+                              {'value_int':0},               # inet ip-mask
+                              {'value_int':0xFFFFFFFF},      # ueaddr ip-mask
+                              {'value_int':0},               # inet port-mask
+                              {'value_int':0},               # ue port-mask
+                              {'value_int':0}])              # proto-id-mask
+
+        FARLookup.delete(fields=[{'value_int':1},                   # far-id
+                              {'value_int':start_teid+i}])          # fseid
+
+        FARLookup.delete(fields=[{'value_int':0},                   # far-id
+                              {'value_int':start_teid+i}])          # fseid
+
+        PreQoSCounter.remove(ctr_id=start_teid+i)                      # fseid
+
+        PostQoSCounter.remove(ctr_id=start_teid+i)                     # fseid
+
+    #bess.reset_all()
+
+
+### Uncomment the following lines to test with il_trafficgen ###
+# Start the test
+#sim_start_test()
+# Wait for 30 seconds before deleting the session info entries (optional)
+#time.sleep(30)
+#sim_end_test()
```

Comment:
Uncomment and guard it with mode == sim

Reply:
Reconsider placement of pause_all/resume_all to account for sim mode? source would have started generating packets by this time.

Reply:
> Reconsider placement of pause_all/resume_all to account for sim mode? source would have started generating packets by this time.

The source is disconnected from the pipeline till all the sessions are added (``sim_start_test``). The source only gets linked up to the pipeline after the call to `sim_start_test()` has been completed. 

`bess.pause_all()/bess.resume_all()` is giving a weird scheduling exception error. Let's discuss this soon.